config/operator.yaml

```diff
@@ -59,6 +59,13 @@ spec:
             periodSeconds: 1
             successThreshold: 1
             timeoutSeconds: 15
+          resources:
+            limits:
+              cpu: "250m"
+              memory: "128Mi"
```

Comment:
Note: I have not tested these values and they're not based on usage metrics in a production environment, so determining the "correct" values will require some judgment from the AuthZed team

![image](https://github.com/authzed/spicedb-operator/assets/52710/1868ba21-8d4e-4f5d-b1c3-d902c37254a4)

Reply:
These values seem good, I can't find deployed spicedb-operator in our production systems that would fall outside this range.

Memory use will scale with the number of managed objects, but 128mb means that even giving a generous 1mb per cluster definition allow over 100 SpiceDBClusters.